package.json

```diff
@@ -2,9 +2,27 @@
   "name": "include-fragment-element",
   "version": "3.0.1",
   "main": "include-fragment-element.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/github/include-fragment-element.git"
+  },
+  "scripts": {
+    "build": "npm install",
+    "clean": "rm -rf ./node_modules",
+    "lint": "jshint *.js test/*.js",
+    "pretest": "npm run lint && npm run build",
```

Comment:
I vote for dropping `npm run build` as part of pretest, since it can take a few seconds each time (even if there's nothing to do) and it doesn't seem really necessary here.